src/main/java/com/google/api/generator/spring/SpringWriter.java

```diff
@@ -139,13 +139,14 @@ private static void writeMetadataFile(GapicContext context, String path, JarOutp
   }
 
   private static void writeSpringFactories(GapicContext context, JarOutputStream jos) {
```

Comment:
Would this mean we need to rebrand this method and the exception message as well? (To something like `writeAutoConfigRegistration`)

Reply:
ah-ha! Good catch.